cmd/kyma/alpha/sign/module/module.go

```diff
@@ -40,6 +40,7 @@ This command signs all module resources recursively based on an unsigned compone
 	cmd.Flags().StringVar(&o.ModPath, "mod-path", "./mod", "Specifies the path where the signed component descriptor will be stored")
 	cmd.Flags().StringVar(&o.SignatureName, "signature-name", "", "name of the signature for signing")
 	cmd.Flags().StringVar(&o.RegistryURL, "registry", "", "Repository context url where unsigned component descriptor located")
+	cmd.Flags().StringVar(&o.NameMappingMode, "nameMapping", "urlPath", "Overrides the OCM Component Name Mapping. Allowed values: \"urlPath\" or \"sha256-digest\". This flag is optional.")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&o.NameMappingMode, "nameMapping", "urlPath", "Overrides the OCM Component Name Mapping. Allowed values: \"urlPath\" or \"sha256-digest\".")
```
The flag is automatically optional since there is a default value, so no need for that in the docs